### DISCUSSION TITLE
Machine: Add "Installed Containers" to overview

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#887] [Suggestions]
Hey, I just came to something that it would be nice to see how many containers are running on a machine. 

I know you can see it, when you install a new server onto one

![image](https://github.com/user-attachments/assets/816bb953-f00b-4ee9-bfba-5623af348ec9)

But maybe above or below the informations from the picture it would be a good nice to have

**Comment by birdman3131** [comment]
It is about 5 pixels above that image next to the name.

![image](https://github.com/user-attachments/assets/46a5349b-ea3a-41f2-93a3-f469459ce2de)

**Reply by birdman3131** [reply]
Ahh. I think we are on slightly different screens. It is shown under https://dash.gameserverapp.com/machine

**Comment by LimitlessDrag0n** [comment]
Yeah when you're on that tab, true 
![image](https://github.com/user-attachments/assets/c781e83c-9bad-4c4d-9de9-588d9c3c9fee)


but thought it might be convienient to have it there as well

![image](https://github.com/user-attachments/assets/b25bedd8-993c-4b42-9143-40a7cdbe0ea6)


but maybe I'm just to lazy to go 2 steps xD
